bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_2jets/Wgamma_Monophoton_Ptg130_2jets_run_card.dat

```diff
@@ -0,0 +1,251 @@
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+  0	=  nhel           ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  10	=  ptj        ! minimum pt for the jets 
+  0	=  ptb        ! minimum pt for the b 
+  130	=  pta        ! minimum pt for the photons 
+  0	=  ptl        ! minimum pt for the charged leptons 
+  0	=  misset     ! minimum missing Et (sum of neutrino's momenta)
+  0	=  ptheavy    ! minimum pt for one heavy final state
+  1.0	=  ptonium    ! minimum pt for the quarkonium states
+  -1	=  ptjmax     ! maximum pt for the jets
+  -1	=  ptbmax     ! maximum pt for the b
+  -1	=  ptamax     ! maximum pt for the photons
+  -1	=  ptlmax     ! maximum pt for the charged leptons
+  -1	=  missetmax  ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0	=  ej      ! minimum E for the jets 
+  0	=  eb      ! minimum E for the b 
+  0	=  ea      ! minimum E for the photons 
+  0	=  el      ! minimum E for the charged leptons 
+  -1	=  ejmax  ! maximum E for the jets
+  -1	=  ebmax  ! maximum E for the b
+  -1	=  eamax  ! maximum E for the photons
+  -1	=  elmax  ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  5	=  etaj     ! max rap for the jets 
+  -1	=  etab     ! max rap for the b
+  3.5	=  etaa     ! max rap for the photons 
+  -1	=  etal     ! max rap for the charged leptons 
+  0.6	=  etaonium  ! max rap for the quarkonium states
+  0	=  etajmin  ! min rap for the jets
+  0	=  etabmin  ! min rap for the b
+  0	=  etaamin  ! min rap for the photons
+  0	=  etalmin  ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+  0.001	=  drjj     ! min distance between jets 
+  0	=  drbb     ! min distance between b's 
+  0.5	=  drll     ! min distance between leptons 
+  0.5	=  draa     ! min distance between gammas 
+  0	=  drbj     ! min distance between b and jet 
+  0.5	=  draj     ! min distance between gamma and jet 
```

Comment:
Do we need this cut? Not better to loosen things a little bit?

Reply:
Removed